template/content/endpoints.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+module.exports = Endpoints;
+
+function Endpoints(app) {
+    if (!(this instanceof Endpoints)) {
+        return new Endpoints(app);
+    }
+
+    var self = this;
+
+    var channel = app.clients.appChannel;
+    var thrift = app.clients.tchannelThrift;
+
+    thrift.register(
+        channel, 'MyService::health_v1', app, self.health
+    );
+    thrift.register(
+        channel, 'MyService::get_v1', app, self.get
+    );
+    thrift.register(
+        channel, 'MyService::put_v1', app, self.put
+    );
+}
+
+Endpoints.prototype.health =
+function health(app, req, head, body, cb) {
```

Comment:
Agreed that it's unlikely that devs will put all handler implementations in a single file. @Willyham are you thinking about one file per endpoint handler?